src/context/index.ts

```diff
@@ -184,10 +191,14 @@ export class Context {
 
     try {
       const blockHash = await system.blockHash(numberToU32(0, this));
-      const balance = await requestAtBlock(balances.totalIssuance, {
-        args: [],
-        blockHash,
-      });
+      const balance = await requestAtBlock(
+        balances.totalIssuance,
+        {
+          args: [],
+          blockHash,
+        },
+        this
```

Comment:
This will throw an error because `isArchiveNode` is false when this is called for the first time